ui/src/overview.tsx

```diff
@@ -40,12 +40,12 @@ import { isValid } from "util/valid";
 import {
   generateCohortFilter,
   getCriteriaPlugin,
-  groupFilterKindLabel,
   groupName,
+  uiGroupFilterFromFilter,
 } from "./cohort";
 
 export function Overview() {
-  const [showDemographics, setShowDemographics] = useState(true);
+  const [showDemographics, setShowDemographics] = useState(false);
```

Comment:
What does this change do -- what state are you no longer using?

Reply:
This was just changing the default. Per the review I'll revert this back to defaulting open though.